aws/s3/Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: build
```

Comment:
The `build` target isn't `PHONY` here.

Reply:
This was just copied from the other makefiles we have in the repo. I'll remove it across the board